src/container/cli.c

```diff
@@ -936,6 +936,9 @@ cont_query_bits(daos_prop_t *prop)
 		case DAOS_PROP_CO_DEDUP_THRESHOLD:
 			bits |= DAOS_CO_QUERY_PROP_DEDUP_THRESHOLD;
 			break;
+		case DAOS_PROP_CO_ALLOCED_OID:
+			bits |= DAOS_CO_QUERY_PROP_MAX_OID;
```

Comment:
Prone to error ...

Reply:
yea i think i missed a few places to change the name, i will push a follow on PR